File: Python sucht Hamster/src/game/GameTimer.java
package game;

public class GameTimer {

	private static double gameDuration; // time in seconds until game is over
	private static int amountImages = 89; // amount of different images for animation
	private static int gameTime; // image that is to be displayed currently

	/*
	 * When making a new GameTimer, set gameTime to 1
	 */
	public GameTimer() {
		GameTimer.gameTime = 1;
	}

	public static int getGameTime() {
		return gameTime;
	}

	public static void setGameTime(int gameTime) {
		GameTimer.gameTime = gameTime;
	}

	public static double getGameDuration() {
		return gameDuration;
	}

	public static void setGameDuration(double gameDuration) {
		GameTimer.gameDuration = gameDuration;
	}

	public static int getAmountImages() {
		return amountImages;
	}

}

File: Python sucht Hamster/C# (Monogame)/game/GameTimer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Python3.game
{
    class GameTimer
    {
        public static double GameDuration; // time in seconds until game is over
        public static int AmountImages = 89; // amount of different images for animation
        public static int GameTime; // image that is to be displayed currently
        public static bool IsModified;

        // When making a new GameTimer, set GameTime to 1 and IsModified to false
        public GameTimer()
        {
            GameTime = 1;
            IsModified = false;
        }
    }
}
